main.py

```diff
@@ -11,7 +17,40 @@
     pass
 
 if __name__ == '__main__':
-    # This is used when running locally only. When deploying to Google App
-    # Engine, a webserver process such as Gunicorn will serve the app. This
-    # can be configured by adding an `entrypoint` to app.yaml.
-    app.run(host='127.0.0.1', port=8080, debug=True)
+    setup_i18n()
+
+    parser = argparse.ArgumentParser(prog='rdr-service', description="RDR web service")
+    parser.add_argument("--debug", help="enable debug output", default=False, action="store_true")  # noqa
+    parser.add_argument("--flask", help="launch flask app", default=False, action="store_true")  # noqa
+    parser.add_argument("--celery", help="launch celery app worker", default=False, action="store_true")  # noqa
+    parser.add_argument("--service", help="launch supervisor and start web service", default=False,
+                            action="store_true")  # noqa
+    parser.add_argument("--unittests", help="enable unittest mode", default=False, action="store_true")  # noqa
+
+    args = parser.parse_args()
+    env = dict(os.environ)
+    if args.unittests:
+        env["UNITTEST_FLAG"] = "True"
+
+    if args.flask and args.celery:
+        print("Flask and Celery options may not be combined, exiting.")
+        exit(1)
+
+    if args.flask:
+        # This is used when running locally only. When deploying to Google App
+        # Engine, a webserver process such as Gunicorn will serve the app. This
+        # can be configured by adding an `entrypoint` to app.yaml.
+        app.run(host='127.0.0.1', port=8080, debug=True)
+        exit(0)
+
+    if args.celery:
```

Comment:
https://cloud.google.com/appengine/docs/standard/python3/runtime we will have to customize this on a new PR.

Reply:
Agreed